### PR TITLE
Bump openapi-generator to 7.7.0

### PR DESCRIPTION
**Pull request by DerTiedemann**
This PR bumps the used version of openapi-generator to the latest version [7.7.0](). The bump is quite big as seen [here](https://github.com/OpenAPITools/openapi-generator/compare/v6.2.1...v7.7.0) (+2k commits / +5k files changed).
Though this might be a big update it improves on the generated code quite a bit, most of all: The sed replacements fixing the missing imports resulting from [this](https://github.com/OpenAPITools/openapi-generator/issues/14171) are fixed.

### CODE REVIEW

File: generate.sh [hunk 1]
```diff
@@ -23,35 +23,6 @@ find src -type f -exec sed -i '/The version of the OpenAPI document/d' {} \;
...
-#             Self::Message => String::from("message"),
-#             Self::Response => String::from("response"),
-#             Self::Request => String::from("request"),
-#             Self::RequestResponse => String::from("requestResponse"),
-#         }
-#     }
-# }
-
-
-sed -z -i 's/impl ToString for InviteMessageType {\n[ a-zA-Z_\(\)&-\>{\n:=",]*}\n    }\n}//g' src/models/invite_message_type.rs
```
Comment (ariesclark) [inline generate.sh:55]: Can you confirm these patches are no longer necessary?

File: patches/.gitkeep
Comment (ariesclark) [inline patches/.gitkeep:1]: This file is not needed.

**Review by ariesclark — CHANGES_REQUESTED** [review]

File: generate.sh (hunk 1 above)
Comment (DerTiedemann) [reply generate.sh:55]: Sure:
- There are no compilation errors. This issue is now closed: https://github.com/OpenAPITools/openapi-generator/issues/14171
- The display is now generated: https://github.com/vrchatapi/vrchatapi-rust/blob/496172375f13d8c2d12a36739aa72e389816b7e4/src/models/invite_message_type.rs#L26-L36

Though i dont have a piece of code I can test this on, someone else would need to do an integration test.

**Review by ariesclark — APPROVED** [review]

**Comment by ariesclark** [comment]
I just noticed you also need to update in the GitHub Action, can you make a PR changing this line:
https://github.com/vrchatapi/vrchatapi-rust/blob/main/.github/workflows/ci.yaml#L28

**Pull request merged**